packages/usa-in-page-navigation/src/index.js

```diff
@@ -0,0 +1,219 @@
+const once = require("receptor/once");
+const keymap = require("receptor/keymap");
+const behavior = require("../../uswds-core/src/js/utils/behavior");
+const { prefix: PREFIX } = require("../../uswds-core/src/js/config");
+const { CLICK } = require("../../uswds-core/src/js/events");
+
+const IN_PAGE_NAV_CLASS = `${PREFIX}-in-page-nav`;
+const IN_PAGE_NAV_LIST_CLASS = `${IN_PAGE_NAV_CLASS}-list`;
+const IN_PAGE_NAV_ITEM_CLASS = `${IN_PAGE_NAV_CLASS}__item`;
+const IN_PAGE_NAV_LINK_CLASS = `${IN_PAGE_NAV_CLASS}__link`;
+const IN_PAGE_NAV_ANCHOR_CLASS = "usa-anchor-tag";
+
+const IN_PAGE_NAV_LINK = `.${IN_PAGE_NAV_LINK_CLASS}`;
+const IN_PAGE_NAV_ANCHOR = `.${IN_PAGE_NAV_ANCHOR_CLASS}`;
+const IN_PAGE_NAV_TITLE = "On this page";
+const IN_PAGE_NAV = `.${PREFIX}-in-page-nav`;
+const CURRENT_CLASS = `${PREFIX}-current`;
+const MAIN_ELEMENT = "main";
+const SITE_SUBHEADING = "site-subheading";
```

Comment:
Is the class name `.site-subheading` pulled from uswds-site? I don't see any references to it in USWDS so I am wondering if there is a more descriptive name

Reply:
Yes, site-subheading is from uswds-site _uswds-theme-custom-styles.scss line 652 
Should I change it to a style that comes with the in-page nav package?

Reply:
I think this hed should get a BEM class related to the in-page-navigation component